### PR TITLE
Criteo : Add user sync url for criteo

### PR DESCRIPTION
**Pull request by vraybaud**
Update production endpoint.
Setup the redirect url.

### CODE REVIEW

File: static/bidder-info/criteo.yaml [hunk 1]
```diff
@@ -14,3 +14,7 @@ userSync:
   # bidder directly at the email address in this file to ask about enabling user sync.
   supports:
     - redirect
+
+  redirect:
+    url: "https://ssp-sync.criteo.com/user-sync/redirect?gdprapplies={{.GDPR}}&gdpr={{.GDPRConsent}}&ccpa={{.USPrivacy}}&redir={{.RedirectURL}}&profile=230"
+    userMacro: "${CRITEO_USER_ID}"
```
Comment (onkarvhanumante) [inline static/bidder-info/criteo.yaml:20]: image

url isn't redirecting https://ssp-sync.criteo.com/user-sync/redirect?gdprapplies=&gdpr=&ccpa=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dcriteo%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BCRITEO_USER_ID%7D&profile=230

Comment (vraybaud) [reply static/bidder-info/criteo.yaml:20]: Hello, thanks for the review. To be able to redirect to a specific domain, we need a configuration on our side. I authorized the domain you want to redirect too and now it redirects.
I will add a comment to make sure this is clear.

Comment (onkarvhanumante) [reply static/bidder-info/criteo.yaml:20]: thanks @vraybaud.  Redirection is working

**Review by onkarvhanumante — APPROVED** [review]

File: static/bidder-info/criteo.yaml (hunk 1 above)
Comment (gargcreation1992) [reply static/bidder-info/criteo.yaml:20]: working for me as well.

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**